trove/render/html_browse.py

```diff
@@ -22,42 +24,58 @@
 from ._base import BaseRenderer
 
 STABLE_MEDIATYPES = (mediatypes.JSONAPI,)
-UNSTABLE_MEDIATYPES = (mediatypes.TURTLE, mediatypes.JSONLD, mediatypes.JSON,)
+UNSTABLE_MEDIATYPES = (
+    mediatypes.TURTLE,
+    mediatypes.JSONLD,
+    # TODO: below are only for search/index-card views
+    mediatypes.JSON,
+    mediatypes.TAB_SEPARATED_VALUES,
+    mediatypes.COMMA_SEPARATED_VALUES,
```

Comment:
oh, because the mediatype for tsv is `text/tab-separated-values`, while the mediatype for comma-separated values is `text/csv`, and at different times i played with that in different ways -- probably should embrace "tsv"/"csv" for consistency, tho, true